### ISSUE TITLE
Please document license requirments

### ISSUE THREAD

**Issue by hkusulja** [MicrosoftDocs/dynamics365smb-docs#332]
Please document license requirmens for this Setup.
For example:
 - Integration account, does it have to have Dynamics CRM and BC licenses?
 - can the license be removed after initial setup (since this is only API usage)

Also for the Admin account, if used only once, does this user needs to have license for CRM and BC, and can it be technically removed license after initial setup? Is Admin account used for any other purpose later, or the whole account can be deleted later?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 80825306-8948-b99c-be8a-bb94c6d3a226
* Version Independent ID: c6cb993e-1b67-f6a8-294c-25769fdd153f
* Content: [Setting Up User Accounts for Integrating with Common Data Service - Business Central](https://docs.microsoft.com/en-US/dynamics365/business-central/admin-setting-up-integration-with-dynamics-sales)
* Content Source: [business-central/admin-setting-up-integration-with-dynamics-sales.md](https://github.com/MicrosoftDocs/dynamics365smb-docs/blob/live/business-central/admin-setting-up-integration-with-dynamics-sales.md)
* Service: **dynamics365-business-central**
* GitHub Login: @bholtorf
* Microsoft Alias: **bholtorf**

**Comment by hkusulja** [comment]
I believe that "integration account" from the docs, is used to sync the data, using API only, and is also in CRM converted to API-only user. Therefore, CRM license can be removed from the integration account.
But unsure about BC account, can we just put the "SUPER" privilege set and without BC license?

Also what for "admin account " / one-time setup account?

**Comment by rubenseishima** [comment]
Hi,

Apologies for replying so late. After consulting with the Dataverse expert, here are his answers:

Can the Dataverse account with the "System Administrator" security role be removed after the initial integration setup.
* Yes
Must the user with the "System Administrator" security role have a license to both CRM and BC during setup?
* This account needs to be a licensed user on the Dataverse environment, be a System Administrator in the Dataverse environment and global admin on the tenant to which this environment belongs.
* This account doesn’t need a license to Business Central – it will be used only to provision the service principal of the S2S app in the Dataverse tenant, and to do setup tasks in the Dataverse environment.
* After the connection setup is done, this user is no longer needed for the purpose of data sync. Synchronization runs via the auto-generated non-interactive application user who doesn’t need any license at all.
Users with "Integration Administrator" and "Integration User" roles in Dataverse must have a valid license for both BC and Dataverse?
* Integration administrator and Integration user roles should only be used by the Application user that runs the integration. The application user does not need the license assigned.

**Comment by hkusulja** [comment]
please update documentation with mentioned statements, thank you.

**Comment by edupont04** [comment]
We will.

**Comment by edupont04** [comment]
Closing as done.
